minio_service/minio_client.py

```diff
@@ -0,0 +1,80 @@
+from config import *
+from minio import Minio
+from minio.error import S3Error
+
+class MinioInit:
+    @staticmethod
+    def minio_auth(access=ConfMinio.access, secret=ConfMinio.secret):
+        return Minio(
+            'localhost:9000',
+            access_key=access,
+            secret_key=secret,
+            secure=False
+        )
+
+
+class MinioCrud:
+    minio = MinioInit().minio_auth()
+
+    @staticmethod
+    def create_new_bucket(bucket_name=ConfMinio.bucket_name, minio=minio):
+        print(bucket_name)
+        if not minio.bucket_exists(bucket_name):
+            try:
+                minio.make_bucket(bucket_name)
+            except S3Error:
+                raise
+
+    @staticmethod
+    def upload_object_to_bucket(file_name, path_to_file, bucket_name=ConfMinio.bucket_name, minio=minio):
+        try:
+            minio.fput_object(
+                bucket_name,
+                file_name,
+                path_to_file
+            )
+        except S3Error:
+            raise
+
+    @staticmethod
+    def delete_object(object_name, bucket_name=ConfMinio.bucket_name, minio=minio):
+
+        try:
+            minio.remove_object(
+                bucket_name, object_name
+            )
+        except S3Error:
+            raise
```

Comment:
what is the point of doing this if you don't process errors? 

in this case 

```
        try:
            minio.remove_object(
                bucket_name, object_name
            )
        except S3Error:
            raise
```

is same as 

```
minio.remove_object(bucket_name, object_name)
```